server/kv_mock_test.go

```diff
@@ -39,9 +40,15 @@ func (jti jiraTestInstance) GetDisplayDetails() map[string]string {
 func (jti jiraTestInstance) GetUserConnectURL(mattermostUserId string) (string, error) {
 	return "http://jiraTestInstanceUserConnectURL.some", nil
 }
+
 func (jti jiraTestInstance) GetClient(jiraUser JIRAUser) (Client, error) {
-	return nil, errors.New("not implemented")
+	client, err := jira.NewClient(&http.Client{}, "testClient")
+	if err != nil {
+		return nil, err
+	}
+	return newCloudClient(client), nil
```

Comment:
I wasn't sure if we wanted to return a mocked Client; It was a fairly big interface to mock out.

Reply:
Right. If we were able to use the mocking library we use elsewhere, it'd be easier since only the necessary methods would need to be mocked out, but we can't yet (it's a separate issue, having to do with how the code is packaged).

Considering my further comments, I am 0/5 on which way is easier for the time being.

Reply:
One more note on this, reviewing https://github.com/mattermost/mattermost-plugin-msoffice/pull/2 I saw @cpurta use `github.com/jarcoal/httpmock` - on the surface seems like a great tool for the job.